crates/miden-lib/asm/kernels/transaction/api.masm

```diff
@@ -917,23 +917,23 @@ end
 #! Returns the current note's inputs commitment and length.
 #!
 #! Inputs:  [pad(16)]
-#! Outputs: [NOTE_INPUTS_COMMITMENT, num_inputs, pad(11)]
+#! Outputs: [NOTE_PAYLOAD_COMMITMENT, num_inputs, pad(11)]
```

Comment:
Here we should also rename `num_inputs` to `payload_length` or something to that effect. Also update the comment above.